docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
+  initialize: function(options) {
+    console.log('Initialize');
+  },
+
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(options.data); 
```

Comment:
@denar90  for the second example I believe this jsfiddle would be more accurate https://jsfiddle.net/k05dctyt/4/ 

and notice if you tried to change options.data to this.options.data it would not work